src/Components/WebAssembly/WebAssembly.Authentication/src/RemoteAuthenticatorViewCore.cs

```diff
@@ -242,6 +252,10 @@ private async Task ProcessLogInCallback()
                     // is when we are doing a redirect sign in flow.
                     throw new InvalidOperationException("Should not redirect.");
                 case RemoteAuthenticationStatus.Success:
+                    if (OnLogInSucceded.HasDelegate)
+                    {
+                        await OnLogInSucceded.InvokeAsync(result.State);
+                    }
```

Comment:
Normally we'd not bother with the `.HasDelegate` check, since it's safe to `InvokeAsync` the `EventCallback<T>` anyway. It's just a no-op if there's no listener.